README.md

```diff
@@ -1 +1,276 @@
+| name | Name of the application | string | - | yes |
+| namespace | Namespace (e.g. `eg` or `cp`) | string | - | yes |
+| security_groups | List of security group IDs to be allowed to connect to the cluster | list | `<list>` | no |
+| stage | Stage (e.g. `prod`, `dev`, `staging`) | string | - | yes |
+| subnet_ids | Subnet ids | list | - | yes |
+| tags | Additional tags (e.g. map(`BusinessUnit`,`XYZ`) | map | `<map>` | no |
+| vpc_id | VPC ID | string | - | yes |
+| zone_awareness_enabled | Enable zone awareness for Elasticsearch cluster | string | `true` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| domain_arn | ARN of the Elasticsearch domain |
+| domain_endpoint | Domain-specific endpoint used to submit index, search, and data upload requests |
+| domain_hostname | Elasticsearch domain hostname to submit index, search, and data upload requests |
+| domain_id | Unique identifier for the Elasticsearch domain |
+| kibana_endpoint | Domain-specific endpoint for Kibana without https scheme |
+| kibana_hostname | Kibana hostname |
+| security_group_id | Security Group ID to control access to the Elasticsearch domain |
+
+
+
+
+
+## References
+
+For additional context, refer to some of these links. 
+
+- [What is Amazon Elasticsearch Service](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/what-is-amazon-elasticsearch-service.html) - Complete description of Amazon Elasticsearch Service
+- [Amazon Elasticsearch Service Access Control](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-ac.html) - Describes several ways of controlling access to Elasticsearch domains
+- [VPC Support for Amazon Elasticsearch Service Domains](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-vpc.html) - Describes Elasticsearch Service VPC Support and VPC architectures with and without zone awareness
+- [Creating and Configuring Amazon Elasticsearch Service Domains](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-createupdatedomains.html) - Provides a complete description on how to create and configure Amazon Elasticsearch Service (Amazon ES) domains
+- [Kibana and Logstash](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-kibana.html) - Describes some considerations for using Kibana and Logstash with Amazon Elasticsearch Service
+- [Amazon Cognito Authentication for Kibana](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-cognito-auth.html) - Amazon Elasticsearch Service uses Amazon Cognito to offer user name and password protection for Kibana
+- [elasticsearch_domain](https://www.terraform.io/docs/providers/aws/r/elasticsearch_domain.html) - Configuration for `elasticsearch_domain` Terraform resource
+- [elasticsearch_domain_policy](https://www.terraform.io/docs/providers/aws/r/elasticsearch_domain_policy.html) - Configuration for `elasticsearch_domain_policy` Terraform resource
```

Comment:
Terraform reference documentation for the `elasticsearch_domain_policy` resource